File: Simple Java App/app/src/main/java/com/srivathsan/testretrofit/network/response/shows/Results.java
package com.srivathsan.testretrofit.network.response.shows;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Results {

    @SerializedName("shows")
    @Expose
    private List<Show> shows = null;
    @SerializedName("count")
    @Expose
    private Integer count;
    @SerializedName("showorder")
    @Expose
    private List<Integer> showorder = null;

    public List<Show> getShows() {
        return shows;
    }

    public void setShows(List<Show> shows) {
        this.shows = shows;
    }

    public Integer getCount() {
        return count;
    }

    public void setCount(Integer count) {
        this.count = count;
    }

    public List<Integer> getShoworder() {
        return showorder;
    }

    public void setShoworder(List<Integer> showorder) {
        this.showorder = showorder;
    }

}

File: Simple Kotlin App/app/src/main/java/com/srivathsan/testretrofit/network/response/shows/Results.kt
package com.srivathsan.testretrofit.network.response.shows

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Results {

    @SerializedName("shows")
    @Expose
    lateinit var shows: List<Show>
    @SerializedName("count")
    @Expose
    var count: Int? = null
    @SerializedName("showorder")
    @Expose
    var showorder: List<Int>? = null

}